src/overdue.sh

```diff
@@ -1,20 +1,20 @@
-#!/usr/bin/env sh
+#!/usr/bin/bash
```

Comment:
I think invoking bash with the `env` shebang `/usr/bin/env bash` is probably the most portable, yes/no?